### ISSUE TITLE
Loss of transparency when increasing scaling

### ISSUE THREAD

**Issue by JotaJota96** [tsujan/Kvantum#546]
I have installed _kvantum Manager_ in **Kubuntu** to use a theme with transparency. Everything works fine until I increase the _global scale_.
I have read that there are problems when using a non-integer scale factor, but in my case I must increase the scaling, I need everything to be a little bigger (I have vision problems).
Is there a way to maintain transparency when using a non-integer factor?

System information:

- **Operating System:** Kubuntu 20.04
- **KDE Plasma Version:** 5.18.5
- **KDE Frameworks Version:** 5.68.0
- **Qt Version:** 5.12.8
- **Kernel Version:** 5.4.0-40-generic
- **OS Type:** 64-bit

**Comment by tsujan** [comment]
> Is there a way to maintain transparency when using a non-integer factor?

Yes. Kvantum 0.16.0 (the latest release) has it: Unchecck `Kvantum Manager → Configure Active theme → Hacks → Disable translucency with non-integer scaling` (and read the warning dialog).

**Comment by tsujan** [comment]
You might also want to know that the probable artifacts will be very bad — almost intolerable — with the old Qt version you have. They're tolerable only with Qt ≥ 5.14.1.

**Comment by JotaJota96** [comment]
I don't know what you mean by 'artifacts'.
How can I update to Qt 5.14.1. or later?

**Comment by tsujan** [comment]
> I don't know what you mean by 'artifacts'.

I don't know what you mean by this question.

> How can I update to Qt 5.14.1. or later?

Your distro should give it to you. Kubuntu? It's Ubuntu, so you need to wait until its next version comes out and then upgrade everything.

**Comment by JotaJota96** [comment]
> I don't know what you mean by this question.

(sorry for my English)
I don't know what you mean by 'artifacts will be very bad', I don't know what 'artifacts' means.

> Your distro should give it to you. Kubuntu? It's Ubuntu, so you need to wait until its next version comes out and then upgrade everything.

Got it. Thanks for your help.

**Comment by tsujan** [comment]
> I don't know what you mean by 'artifacts will be very bad', I don't know what 'artifacts' means.

With Qt ≤ 5.14 + non-integer scaling + translucency, you might see lines that shouldn't exist or not see lines/curves that should exist, etc. That was why I disabled translucency with non-integer scaling in the first place. But because redundant or missing lines became less frequent with Qt 5.14, I added an option to Kvantum Manager for users who might want to accept the risk.
